scripts/pull_request_blaster_outer/change_ci_ruby_versions.rb

```diff
@@ -18,7 +18,7 @@
 
 changed = false
 
-content = File.read(ci).gsub(/^\s*on:/, "\"on\":")
+content = File.read(ci)
```

Comment:
If we go this route, we could drop this and go back to `yaml = YAML.load_file(ci)`

Reply:
:+1: